src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -26,23 +26,44 @@
     }
 
     @RequestMapping(method = RequestMethod.GET, value = "getTask")
-    public TaskDto getTask(@RequestParam("id") Long id) {
-        return taskMapper.mapToTaskDto(dbService.findById(id));
+    public TaskDto getTask(@RequestParam Long taskId) throws TaskNotFoundException {
```

Comment:
Nie podoba mi si, ze metoda rzuca wyjatek, jak nie ma zadania :/  To sie nazywa sterowanie wyjatkami, co zrobiles. Pomysl o przeniesieniu czesci logiki do metody changeTask. Moglaby zwracac true, jesli uda sie zmienic, a false w przeciwnym razie.